Data/DataModel/TeamMessage.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace ServerCore.DataModel
+{
+    /// <summary>
+    /// A message from a team.
+    /// </summary>
+    public class TeamMessage : Message
+    {
+        public TeamMessage()
+        {
+            this.IsFromGameControl = false;
```

Comment:
Nit: this shouldn't be necessary since bools default to false, but if you want, you could also set the property in the base class

Reply:
That is true. I just wanted to set this explicitly